code/modules/mob/living/simple_animal/abnormality/he/steam_transport_machine.dm

```diff
@@ -0,0 +1,177 @@
+/mob/living/simple_animal/hostile/abnormality/steam
+	name = "Steam Transport Machine"
+	desc = "A bipedal, steam-powered automaton made of a brown, wood-like material with brass edges."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "steam"
+	icon_living = "steam"
+	maxHealth = 1600
+	health = 1600
+	ranged = TRUE
+	attack_sound = 'sound/effects/hit_kick.ogg'
+	friendly_verb_continuous = "bonks"
+	friendly_verb_simple = "bonk"
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 2, PALE_DAMAGE = 1.5)
```

Comment:
Brute doesn't have to be defined in this list anymore.

Reply:
A code improvement to make it standard seems like a good idea